config/symbbase.php

```diff
@@ -48,6 +48,21 @@
 $SYMB_UID = (array_key_exists('uid',$PARAMS_ARR)?$PARAMS_ARR['uid']:0);
 $IS_ADMIN = (array_key_exists('SuperAdmin',$USER_RIGHTS)?1:0);
 
+//Set accessibilty variables
+$ACCESSIBILITY_ACTIVE = false;
+$isAccessiblePreferred = $pHandler->getAccessibilityPreference($SYMB_UID);
+// $_SESSION['active_stylesheet'] = null; // use this if you want to troubleshoot the behavior of just the persisted preference
+$localSession = isset($_SESSION['active_stylesheet']) ? $_SESSION['active_stylesheet'] : null;
+if($isAccessiblePreferred){
+	if(!isset($localSession) || !strpos($localSession, 'condensed.css')){
+		$ACCESSIBILITY_ACTIVE == true;
```

Comment:
Why `==` here?

Reply:
I'm also confused why both of these seem like they ought to set $ACCESSIBILITY_ACTIVE to true. But it seems to be working as expected when I toggle between modes, so I guess it's ok??

Or perhaps it's the `==` bug that's hiding this second bug?